client/components/jobs/JobItem.tsx

```diff
@@ -27,7 +27,7 @@ export default function JobItem() {
     >
       <Box ml="0.5%" maxW="100%">
         <Link
-          href="/jobs/project-1"
+          href="/jobs/username/marketing-manager"
```

Comment:
Not sure if we use these paths anywhere else, but if we do, we should abstract them out and import into each spot so we only have to change it once in a central location

A constants file maybe?

Reply:
I agree with this in theory but I think we should wait till it becomes an actual problem for us, like we should wait till we've had to fix a broken link or three and then we can figure out an abstraction for that. Makes sure we understand the pain(s) we're solving for before we abstract.